services/ymax-planner/src/pending-tx-manager.ts

```diff
@@ -87,6 +87,7 @@ const cctpMonitor: PendingTxMonitor<CctpTx, EvmContext> = {
     const logPrefix = `[${txId}]`;
 
     // Parse destinationAddress format: 'eip155:42161:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092'
+    assert.typeof(destinationAddress, 'string');
```

Comment:
Why make this assertion? without it I see the type as:
```
const destinationAddress: `${string}:${string}:${string}`
```

Maybe you had this commit for another base branch and it's obsolete now?